app/controllers/analytics_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class AnalyticsController < ApplicationController
+  def show
+    resource = FindResource.call(params[:resource_id])
+    view_statistics = load_view_statistics(resource)
+
+    respond_to do |format|
+      format.html { head :unsupported_media_type }
+      format.json { render json: view_statistics }
+    end
+  end
+
+  private
+
+    def load_view_statistics(resource)
+      stats = if resource.is_a?(Work)
```

Comment:
@rschenk what if this was a method on the model? Work#stats and WorkVersion#stats

Reply:
I thought about that. Maybe I could put it inside the ViewStatistics concern module, then override it on Work

Reply:
meh. I'd say leave it for now. up to you...